docs/topics/configmap-mission-deploy-booster.adoc

```diff
@@ -81,4 +113,21 @@ NAME                 HOST/PORT                                     PATH      SER
 +
 A pod's route information gives you the base URL which you use to access it. In the example above, you would use `http://{app-name}-myproject.192.168.99.100.nip.io` as the base URL to access the application.
 
+ifdef::configmap-spring-boot-tomcat[]
```

Comment:
Shouldn't this spring-boot-tomcat config map verification block follow the `mvn clean fabric8:deploy -Popenshift` command (which in case of springboot creates the config map from .yml file and then deploys the application, thus the verification steps should be in that order)?